xmodule/docs/decisions/0005-send-data-to-edx-submission.rst

```diff
@@ -0,0 +1,191 @@
+
+.. code-block:: bash
+
+    cd xqueue-watcher
+    pip install -r requirements.txt
+
+**Configuration:**
+
+1. Locate the configuration file at:
+
+   ::
+
+     config/conf.d/course.json
+
+2. Update the `course.json` file with the following configuration:
+
+.. code-block:: json
+
+    {
+      "test-123": {
+        "SERVER": "http://127.0.0.1:18040",
+        "CONNECTIONS": 1,
+        "AUTH": ["username", "password"]
+      }
+    }
+
+- **test-123**: The name of the queue to listen to.
+- **SERVER**: The Xqueue server address.
+- **AUTH**: A list containing `[username, password]` for Xqueue Django user authentication.
+- **CONNECTIONS**: Number of threads to spawn to watch the queue.
+
+3. Start the Xqueue-Watcher service:
+
+.. code-block:: bash
+
+    python watcher.py
+
+3. Setting up Xqueue-Submission
+-------------------------------
+
+This new flow sends queue data to **edx-submission** for processing.
+
+**Steps:**
+
+1. Create a new instance of Xqueue-Watcher:
+
+.. code-block:: bash
+
+    git clone https://github.com/openedx/xqueue-watcher.git
+
+2. Configure the new instance to listen on port **8000**. Edit the `course.json` file located at:
+
+   ::
+
+     config/conf.d/course.json
+
+3. Add the following configuration:
+
+.. code-block:: json
+
+    {
+      "test-123": {
+        "SERVER": "http://127.0.0.1:8000",
+        "CONNECTIONS": 1,
+        "AUTH": ["username", "password"]
+      }
+    }
+
+- **SERVER**: Now points to port **8000**, where edx-submission is running.
+
+4. Start the new instance of Xqueue-Watcher:
+
+.. code-block:: bash
+
+    python watcher.py
+
+4. Verification
+---------------
+
+1. Ensure that all services are running:
+   - Verify that ports **18040** and **8000** are active.
+   - Check the logs for connection errors or authentication issues.
+
+2. Test the configuration:
+   - Send a test request to the queue `test-123` to confirm data processing through **edx-submission**.
+
+
+Consequences
+************
+
+**Positives:**
+
+- **Enhanced Submission Handling:** Transitioning to the `edx-submission` service offers a more modern and efficient architecture for processing student responses, improving scalability and maintainability.
+
+- **Controlled Migration:** The temporary *waffle flag* allows for a phased transition, enabling administrators to test and adopt the new submission process on a per-course basis, thereby minimizing potential disruptions.
+
+**Negatives:**
+
+- **Increased Complexity:** Introducing a temporary flag-based flow adds complexity to the codebase, which may increase maintenance efforts during the transition period.
+
+
+References
+**********
+
+- **Relevant commits**: [Implementation of the Waffle Flag and modification of xqueue_interfaces.py](https://github.com/aulasneo/edx-platform/commit/f50afcc301bdc3eeb42a 6dc2c051ffb2d799f868#diff-9b4290d2b574f54e4eca7831368727f7ddbac8292aa75ba4b28651d4bf2bbe6b)
```

Comment:
This is RST, not markdown. See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Also I am not sure it's appropriate to link to a single commit on a fork. Linking to this PR may be more appropriate.